msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards are known for their ability to adapt to the host application's overall styling. However, their adaptability is limited when it comes to the layout. The static layout approach results in unsatisfactory user experiences, especially across different screen sizes and resolutions.
+
+Adaptive Card responsive layout allows you to design Adaptive Cards that adapt to different screen sizes in Teams. Responsive Adaptive Cards dynamically adjust their layout based on the available screen width, ensuring optimal display regardless of the device or screen size. When you design an Adaptive Card for your app, use the `targetWidth` property to make your responsive and have a better user experience on Teams mobile or desktop, regardless of the screen size.
+
+#### Configure Adaptive Card responsive layout
+
+The following are the key elements to create a responsive Adaptive Card:
+
+* `CardElement.targetWidth`: Allows you to control the visibility of any element by linking it to `hostWidth`. Add the `targetWidth` property for all elements in an Adaptive Card to define different layouts for different card widths. The following are the predefined values for the `targetWidth` property:
```

Comment:
`hostWidth` is NOT to be mentioned in our documentation. It is an implementation detail that is irrelevant to developers. There is no `hostWidth` property a developer can read or otherwise refer to. The only thing there is is "the width of the card" which can be verry narrow, narrow, standard or wide.

When designing a responsive card, a developer can decide that certain elements are only vbisible when "the width of the card" is "very narrow" (for example). To do that, they would set `"targetWidth": "veryNarrow"` on that element.

Reply:
What you really want to do is detail when the developer should expect their cards to have a certain width:
- On a mobile phone in portrait mode, the card's width is "narrow"
- On a mobile phone in landscape mode, the card's width is "standard"
- On a tablet in portrait mode, the card's with is "standard"
- On a tablet in landscape mode, the card's width is "wide"
- In the meeting chat pane on desktop, the card's width is "verryNarrow"
- In the normal chat pane on desktop, the card's width can go from "standard" to "wide"
- In a channel; on desktop, the card's width is "wide"

Then you can go on and explain that setting the `targetWidth` property of an element will make that element on show when their card has that width. The table below doesn't really explain much.